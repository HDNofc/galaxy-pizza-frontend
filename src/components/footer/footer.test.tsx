import renderWithThemeProvider from 'test-utils/render-with-theme-provider';
import Footer from './footer';

describe('<Footer>', () => {
  it('отображается корректно', () => {
    const { container } = renderWithThemeProvider(<Footer />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
