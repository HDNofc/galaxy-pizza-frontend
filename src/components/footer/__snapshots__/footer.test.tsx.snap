// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer> отображается корректно 1`] = `
<div
  class="Footer-sc-1sq4u4p-0 cla-dGb"
>
  <div
    class="Content-sc-1sq4u4p-1 kFlCIn"
  >
    <span
      class="Copyright-sc-1sq4u4p-4 bNGocS"
    >
      © Copyright
    </span>
    <div
      class="MenuFooter-sc-1sq4u4p-3 eUEzlh"
    >
      <a
        class="NativeLink-sc-16ffk5r-1 tmBDA ContactLink-sc-1sq4u4p-2"
        href="tel:+78009999999"
      >
        8 (800) 999-99-99
      </a>
      <a
        class="NativeLink-sc-16ffk5r-1 tmBDA ContactLink-sc-1sq4u4p-2"
        href="mailto:mail@example.com"
      >
        mail@example.com
      </a>
    </div>
  </div>
</div>
`;
